blackrock/templates/portal/webcam.html

```diff
@@ -62,16 +62,16 @@
             <center>
                 <!-- Sunrise -->
                 <div id="sunrise" style="display: none;" class="video">
-                    <script  type="text/javascript" src="https://ccnmtl.columbia.edu/stream/jsembed?player=download_mp4_v3&file=projects/mp4/virtualforest_time_lapse01_glare.mp4&width=480&height=360&poster=https://ccnmtl.columbia.edu/broadcast/projects/mp4/virtualforest_time_lapse01_glare.jpg&protection=5b9e4fe881b47f0a4ffcaa630dc4f0ca4d36d1e1"></script>
+                    <script  type="text/javascript" src="https://www1.ccnmtl.columbia.edu/sec/ccnmtl/remote/static/stream/jsembed?player=download_mp4_v3&file=projects/mp4/virtualforest_time_lapse01_glare.mp4&width=480&height=360&poster=https://ccnmtl.columbia.edu/broadcast/projects/mp4/virtualforest_time_lapse01_glare.jpg&protection=5b9e4fe881b47f0a4ffcaa630dc4f0ca4d36d1e1"></script>
                 </div>    
                 
                 <!-- Snow Melting -->
                 <div id="snow" style="display: none" class="video">
-                    <script type="text/javascript" src="https://ccnmtl.columbia.edu/stream/jsembed?player=download_mp4_v3&file=projects/mp4/virtualforest_time_lapse03_snow_long.mp4&width=480&height=360&poster=https://ccnmtl.columbia.edu/broadcast/projects/mp4/virtualforest_time_lapse03_snow_long.jpg&protection=c0a8e54641fecf4f2d2daf1ffe96fc5b83c28dca"></script>
+                    <script type="text/javascript" src="https://www1.ccnmtl.columbia.edu/sec/ccnmtl/remote/static/stream/jsembed?player=download_mp4_v3&file=projects/mp4/virtualforest_time_lapse03_snow_long.mp4&width=480&height=360&poster=https://ccnmtl.columbia.edu/broadcast/projects/mp4/virtualforest_time_lapse03_snow_long.jpg&protection=c0a8e54641fecf4f2d2daf1ffe96fc5b83c28dca"></script>
                 </div>
                 <!-- April Showers -->
                 <div id="showers" style="display: none" class="video">
-                    <script type="text/javascript" src="https://ccnmtl.columbia.edu/stream/jsembed?player=download_mp4_v3&file=projects/mp4/virtualforest_time_lapse02_april_showers.mp4&width=480&height=360&poster=https://ccnmtl.columbia.edu/broadcast/projects/mp4/virtualforest_time_lapse02_april_showers.jpg&protection=3bf74c25d2c49a56a6d6c9612134a2d25fb3e1c1"></script>
+                    <script type="text/javascript" src="https://www1.ccnmtl.columbia.edu/sec/ccnmtl/remote/static/stream/jsembed?player=download_mp4_v3&file=projects/mp4/virtualforest_time_lapse02_april_showers.mp4&width=480&height=360&poster=https://ccnmtl.columbia.edu/broadcast/projects/mp4/virtualforest_time_lapse02_april_showers.jpg&protection=3bf74c25d2c49a56a6d6c9612134a2d25fb3e1c1"></script>
```

Comment:
Thanks! I never touched that module so I'm a bit hesitant to change too many things at once. For now I'll just change the links back.